docs/contributing-guide.md

```diff
@@ -0,0 +1,90 @@
+# Contributing guide
```

Comment:
The only advantage we have is that it adds a link when you submit an issue that nobody reads, I prefer to keep it here for consistency.